core/modules/qdisp/QueryRequest.cc

```diff
@@ -66,41 +67,43 @@ class QueryRequest::AskForResponseDataCmd : public PriorityCommand {
     void action(util::CmdData *data) override {
         // If everything is ok, call GetResponseData to have XrdSsi ask the worker for the data.
         {
+            util::InstanceCount _ic{"1-act-AskForResponseDataCmd"};
             auto jq = _jQuery.lock();
             auto qr = _qRequest.lock();
             if (jq == nullptr || qr == nullptr) {
-                LOGS(_log, LOG_LVL_WARN, _idStr << " AskForResponseData null before GetResponseData");
+                LOGS(_log, LOG_LVL_WARN, _idStr << " AskForResp null before GetResponseData");
                 // No way to call _errorFinish().
                 _setState(State::DONE2);
                 return;
             }
 
             if (qr->isQueryCancelled()) {
-                LOGS(_log, LOG_LVL_DEBUG, _idStr << " AskForResponseData query was cancelled");
+                LOGS(_log, LOG_LVL_DEBUG, _idStr << " AskForResp query was cancelled");
                 qr->_errorFinish(true);
                 _setState(State::DONE2);
                 return;
             }
             std::vector<char>& buffer = jq->getDescription()->respHandler()->nextBuffer();
-            LOGS(_log, LOG_LVL_DEBUG, _idStr << " Asking for GetResponseData size=" << buffer.size());
+            LOGS(_log, LOG_LVL_DEBUG, _idStr << " AskForResp GetResponseData size=" << buffer.size());
             qr->GetResponseData(&buffer[0], buffer.size());
         }
 
         // Wait for XrdSsi to call ProcessResponseData with the data,
         // which will notify this wait with a call to receivedProcessResponseDataParameters.
         {
+            util::InstanceCount _ic{"2-act-AskForResponseDataCmd"};
```

Comment:
Same comment on the name of the local variable which should not start with underscore.